src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final int VALID_USER_AGE = 19;
+    private static final String VALID_USER_LOGIN = "MyLongLogin";
+    private static final String VALID_USER_PASSWORD = "MyLongPassword";
+    private static final int INVALID_USER_AGE = 2;
+    private static final String INVALID_USER_LOGIN = "login";
+    private static final String INVALID_USER_PASSWORD = "pass";
+    private static final int DUPLICATE_VALID_USER_AGE = 20;
+    private static final String DUPLICATE_VALID_USER_LOGIN = "LongLogin";
+    private static final String DUPLICATE_VALID_USER_PASSWORD = "LongPassword";
+    private static final int NEGATIVE_USER_AGE = -10;
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    void register_userValid_Ok() {
```

Comment:
```suggestion
    void register_userValid_ok() {
```
lowercase after underscore